x-pack/plugins/observability_ai_assistant/server/functions/kibana.ts

```diff
@@ -64,6 +64,9 @@ export function registerKibanaFunction({
         query,
       };
 
+      resources.logger.info(JSON.stringify(request.url));
+      resources.logger.info(JSON.stringify(request.headers, null));
```

Comment:
Maybe debug?

Reply:
I need to inspect the cloud deployment. I'll remove the statements entirely (because this leaks security details 😛)